src/ledger/fuzz.zig

```diff
@@ -0,0 +1,330 @@
+const std = @import("std");
+const sig = @import("../sig.zig");
+
+const ColumnFamily = sig.ledger.database.ColumnFamily;
+const AtomicU64 = std.atomic.Value(u64);
+
+var total_action_count: AtomicU64 = AtomicU64.init(0);
+
+const allocator = std.heap.c_allocator;
+
+const Data = struct {
+    value: []const u8,
+};
+
+const cf1 = ColumnFamily{
+    .name = "data",
+    .Key = u64,
+    .Value = Data,
+};
+pub const BlockstoreDB = switch (build_options.blockstore_db) {
+    .rocksdb => ledger.database.RocksDB(&.{cf1}),
+    .hashmap => ledger.database.SharedHashMapDB(&.{cf1}),
+};
+
+const build_options = @import("build-options");
```

Comment:
can we keep all imports at the top

Reply:
https://github.com/Syndica/sig/pull/452/commits/c7efe51d70e31b6f8575e2298fa7793946060cb4